test/obb/events/game_events_test.clj

```diff
@@ -21,29 +21,20 @@
   (let [system (-> (game-events-system) component/start)
         game-events (:game-events system)
         game-info (game-info/create)
-        subscriber-ch (async/chan 1)
+        game-ch (async/chan 1)
         subscriber-all-ch (async/chan 1)
         test-event :new-game
```

Comment:
The `test-event` binding. I suppose it was used with the previous event model, right?